build/configs/esp32_DevKitC/scripts/esp32.template

```diff
@@ -47,7 +47,7 @@ MEMORY
 
   /* Even though the segment name is iram, it is actually mapped to flash */
 
-  iram0_2_seg (RX) :     org = 0x400D0018, len = 0x330000
+  iram0_2_seg (RX) :     org = 0x400D0018, len = 0x330000-0x18
```

Comment:
Why? I can't find why you modify like this at anywhere.

Reply:
okay,  added  comment for modifications